sql/core/pom.xml

```diff
@@ -265,6 +297,36 @@
           </execution>
         </executions>
       </plugin>
+      <!-- Add protobuf-maven-plugin and provide ScalaPB as a code generation plugin -->
```

Comment:
@grundprinzip Need to consider compatibility with Apple Silicon

Reply:
Yeah, I had to find a colleague with M1 to test, I pushed a new version that actually works.